### DISCUSSION TITLE
about   VSCode command : vue (Debug): Write Virtual Files

### DISCUSSION THREAD

**Discussion by well-he** [vuejs/language-tools#4163] [Q&A] [answered]
i run  VSCode command VSCode command : vue (Debug): Write Virtual Files 。 What should I do to delete the generated files quicky? it makes my git repo dirty 。

**Comment by so1ve** [comment, marked as answer]
Just use a glob to delete them. This command is designed for volar maintainers.

**Reply by well-he** [reply]
thx, I deleted these files using command in shell

**Reply by well-he** [reply]
windows powershell : `Get-ChildItem -Filter *.vue.js -Recurse | Remove-Item `
